### ISSUE TITLE
Need to migrate to Drive API before April 20, 2015

### ISSUE THREAD

**Issue by devotis** [theoephraim/node-google-spreadsheet#21]
Google recently [communicated](http://googleappsupdates.blogspot.nl/2014/12/reminder-to-migrate-to-updated-google.html) that Documents List API will be discontinued on April 20, 2015. This change means that service calls for this API will no longer be supported, and any Google Apps features that are implemented using this API will no longer function.

One of the URL's that will be discontinued is https://spreadsheets.google.com/feeds/ which is used by this module. It's suggested to migrate to the [Drive API](https://developers.google.com/drive/).

To summarize, node-google-spreadsheet will no longer function as of April 20, 2015 if no action is taken.

**Comment by theoephraim** [comment]
thanks for the heads up!

I'll be gone travelling all of February, but if no one else jumps in to tackle this by the time I'm back, I'll get on it in mid March!

**Comment by devotis** [comment]
Update on the matter. You don't need to migrate. Read here:
https://github.com/jpillora/node-edit-google-spreadsheet/issues/57
